website/docs/d/healthcare_dicom.html.markdown

```diff
@@ -51,6 +63,29 @@ An `authentication` supports the following:
 
 * `audience` - The intended audience to receive authentication tokens for the service. The default value is <https://dicom.azurehealthcareapis.azure.com>
 
+---
+
+A `cors` supports the following:
```

Comment:
For data sources we say a block `exports` rather than `supports` since these aren't arguments that can be supplied 
```suggestion
A `cors` exports the following:
```

Reply:
Updated